helm_deploy/values-dev.yaml

```diff
@@ -0,0 +1,25 @@
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 2
+
+image:
+  repository: quay.io/hmpps/offender-assessment-events
+  tag: latest
+  port: 8080
+
+ingress:
+  enabled: true
+  enable_whitelist: true
+  hosts:
+    - host: offender-assessment-events-dev.hmpps.service.justice.gov.uk
```

Comment:
This needs to be different for the AKS cluster, it won't be this address, and the name offernder-assessment-events-dev will be too long for the 64 char limit.